### ISSUE TITLE
Unhandled exception (#5a7f47b0)

### ISSUE THREAD

**Issue by sqlmapreporter** [sqlmapproject/sqlmap#3700]
```
Running version: 1.3.5.135#dev
Python version: 2.7.16
Operating system: Linux-4.4.153-perf+-aarch64-with-libc
Command line: sqlmap.py -v 2 --tor -u ********************************************************* --null-connection --level 5 --risk 2 --sqlmap-shell --random-agent --drop-set-cookie --delay 0.01 --time-sec 10 --tamper randomcase,varnish,xforwardedfor,between,multiplespaces.py --threads 10
Technique: UNION
Back-end DBMS: MySQL (fingerprinted)
```
```
Traceback (most recent call last):
  File "sqlmap.py", line 180, in main
    start()
  File "lib/core/decorators.py", line 71, in _
    result = f(*args, **kwargs)
  File "lib/controller/controller.py", line 673, in start
    action()
  File "lib/controller/action.py", line 165, in action
    conf.dbmsHandler.dumpAll()
  File "plugins/generic/entries.py", line 500, in dumpAll
    self.dumpTable()
  File "plugins/generic/entries.py", line 453, in dumpTable
    conf.dumper.dbTableValues(kb.data.dumpedTable)
  File "lib/core/dump.py", line 605, in dbTableValues
    self._write("| %s%s" % (value, blank), newline=False, console=console)
  File "lib/core/dump.py", line 79, in _write
    dataToStdout(text)
  File "lib/core/common.py", line 975, in dataToStdout
    sys.stdout.write(stdoutEncode(setColor(data, bold=bold)))
  File "/usr/lib/python2.7/codecs.py", line 369, in write
    data, consumed = self.encode(object, self.errors)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xd2 in position 52: ordinal not in range(128)
```

**Comment by huitc** [comment]
`--hex --dump-all --exclude-sysdbs --batch --stop 3 --tor`

**Comment by stamparm** [comment]
Root cause should be fixed with #3702
